public/app/features/dashboard-scene/utils/dashboardSessionState.ts

```diff
@@ -1,42 +1,47 @@
 import { UrlQueryMap, urlUtil } from '@grafana/data';
 import { config, locationService } from '@grafana/runtime';
-import { UrlSyncManager } from '@grafana/scenes';
 
 import { DashboardScene } from '../scene/DashboardScene';
 
 export const PRESERVED_SCENE_STATE_KEY = `grafana.dashboard.preservedUrlFiltersState`;
 
 export function restoreDashboardStateFromLocalStorage(dashboard: DashboardScene) {
-  if (!dashboard.state.version) {
```

Comment:
Mhm what if we do something like `if (!isNumber(dashboard.state.version)) { return; }` here?